src/easy.rs

```diff
@@ -2864,6 +2951,11 @@ impl Easy {
         }
     }
 
+    fn getopt_double(&mut self, opt: curl_sys::CURLINFO) -> Result<c_double, Error> {
+        self.getopt_ptr(opt)
+            .map(|p| unsafe {transmute::<_, c_double>(p)})
```

Comment:
I am not really sure what you mean by calling the FFI function directly.
How does one go from *const i8 to f64 in a portable way?